tests/test_e2e.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import multiprocessing
+import time
+
+import pytest
+import requests
+import uvloop
+from vllm.entrypoints.openai.api_server import (
+    make_arg_parser, run_server, validate_parsed_serve_args)
+from vllm.utils import FlexibleArgumentParser
+
+
+def _run_server_process(args):
+    uvloop.run(run_server(args))
+
```

Comment:
```suggestion

@pytest.fixture(scope="module")
def model_name():
    return "meta-llama/Llama-3.1-8B-Instruct"
```

Make this a fixture so it's easy to change which model we test!

Reply:
This is parametrized now